packages/android_intent/android/src/test/java/io/flutter/plugins/androidintent/MethodCallHandlerImplTest.java

```diff
@@ -283,4 +285,38 @@ public void onMethodCall_setsDataAndType() {
     assertEquals(type, intent.getType());
     assertEquals(data, intent.getData());
   }
+
+  @Test
+  public void onMethodCall_setsIgnoredPackages() {
```

Comment:
Can you describe what this is testing? As far as I can tell, this test is verifying that including ignored packages doesn't impact the result at all. But isn't it also possible that sometimes the ignored packages will impact the behavior? Shouldn't there a best for that, too?

Reply:
I would say that it tests that the package that was added to the `ignoredPackages` is not used to resolve the intent.
I will try to update the test to explicitly verify that the specified package to ignore was excluded.